test/sdf/empty_road_sph_coords.sdf

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" ?>
+<sdf version="1.7">
+  <world name="default">
+    <road name="empty_road"/>
+     <spherical_coordinates/>
```

Comment:
Nit: extra white space in front.

Reply:
Fixed at: https://github.com/osrf/sdformat/pull/542/commits/64f226fb26cfae7810fbb5114b29cb1ad8f47279